src/containers/CreatePipelineRun/CreatePipelineRun.js

```diff
@@ -318,15 +318,20 @@ class CreatePipelineRun extends React.Component {
   };
 
   initialState = () => {
-    const { namespace, pipelineRef } = this.props;
+    const { namespace } = this.props;
+    let { pipelineRef } = this.props;
     const pipelineInfo = parsePipelineInfo(pipelineRef, namespace);
+    if (pipelineInfo.pipelineError === true) {
```

Comment:
It's a nit but you need to fix the commit message typo as well please (go with the description I've got for this PR), can this just be ` if (pipelineInfo.pipelineError === true) {`?

Otherwise all good